tool/gravity/cli/operation.go

```diff
@@ -182,6 +182,17 @@ func rollbackPlan(localEnv *localenv.LocalEnvironment, environ LocalEnvironmentF
 	default:
 		return trace.BadParameter(unsupportedRollbackWarning, op.TypeString())
 	}
+
+	statusList, err := collectAgentStatus(localEnv)
+	if err != nil {
+		return trace.Wrap(err, "failed to collect agent status")
+	}
+
+	if !statusList.AgentsActive() {
+		localEnv.Println(statusList.String())
+		return trace.BadParameter("some agents are offline; ensure all agents are deployed with `gravity agent deploy`")
```

Comment:
I would update the message to say `./gravity agent deploy` to indicate that it should be run with the binary from the upgrade tarball.